### ISSUE TITLE
New page is not created

### ISSUE THREAD

**Issue by cebreus** [anchorcms/anchor-cms#1208]
### Summary

Unable to create a new page.

### Expected Behaviour

Be able to create a new page.

### Actual Behaviour

- Log in admin > open the _Pages_ (/admin/pages/) > click on the button _Create a new page_ (/admin/pages/add).
- New page is not created.

![01-bworser-with-console](https://user-images.githubusercontent.com/1650647/27511258-39b9d872-5920-11e7-88e7-a0a951497c3a.png)

### Context details (if applicable)

- Anchor version: 0.12.3
- Server setup: wampserver on windows 64bit; PHP 7.0.10
- errors.log `{"date":"2017-06-24 20:54:13","message":"Undefined variable: id","trace":[{"file":"C:\\wamp64\\www\\anchor-cms\\anchor\\routes\\pages.php","line":222,"function":"native","class":"System\\error","type":"::","args":[8,"Undefined variable: id","C:\\wamp64\\www\\anchor-cms\\anchor\\routes\\pages.php",222,{"input":{"parent":"0","name":"New page","title":"New page","slug":"new-page","markdown":"","status":"published","redirect":"","show_in_menu":"","pagetype":"all"},"dont_encode":["markdown"],"key":"pagetype","value":"all","validator":{},"errors":{"slug":["Slug already exists"]}}]},{"function":"{closure}","args":[]},{"file":"C:\\wamp64\\www\\anchor-cms\\system\\route.php","line":165,"function":"call_user_func_array","args":[{},[]]},{"file":"C:\\wamp64\\www\\anchor-cms\\system\\router.php","line":158,"function":"run","class":"System\\route","type":"->","args":[]},{"file":"C:\\wamp64\\www\\anchor-cms\\system\\start.php","line":46,"function":"dispatch","class":"System\\router","type":"->","args":[]},{"file":"C:\\wamp64\\www\\anchor-cms\\index.php","line":35,"args":["C:\\wamp64\\www\\anchor-cms\\system\\start.php"],"function":"require"}],"line":222,"file":"C:\\wamp64\\www\\anchor-cms\\anchor\\routes\\pages.php"}
`

**Comment by TheBrenny** [comment]
Oh whoops! What an interesting catch! I'm unable to get to this one now or during the week - if someone can, start here: [anchor/routes/pages.php#L222](https://github.com/anchorcms/anchor-cms/blob/master/anchor/routes/pages.php#L222)

I think we should just be able to take this out, no?

**Comment by Radiergummi** [comment]
Should be enough to replace `'id' => $id` with `'id' => $page->id`, shouldn't it?

**Comment by cebreus** [comment]
Another interesting connection. A new page can not be created even if the (new) 'slug url' is already used. After removing the messages, this information can not be completed. I did not find it until I installed an older version of Anchor CMS that had these messages.

I think it would be advisable to return debug and status messages at least as a listing in the browser console as console.log or console.error.

Thank for consideraion.
